atoma-proxy/src/server/chat_completions.rs

```diff
@@ -230,6 +255,21 @@ async fn handle_non_streaming_response(
         return Err(StatusCode::INTERNAL_SERVER_ERROR);
     }
 
+    state
+        .state_manager_sender
+        .send(
+            AtomaAtomaStateManagerEvent::UpdateNodeThroughputPerformance {
+                node_small_id: node_id,
+                input_tokens,
+                output_tokens,
+                time: time.elapsed().as_secs_f64(),
```

Comment:
This is not a good metric of throughput, I reckon. We should separate the time to first token (prefill phase, a.k.a in-context processing) and once we have that value, we can update the time from first token to the end of the chat stream (decoding phase). We can also measure latency as the average/median (and possibly stddev) of time per token.

That said, we can only extract this refined measures, if the result is being streamed back to the proxy.